test/Chirp.Infrastructure.Tests/UnitTest.cs

```diff
@@ -33,16 +36,6 @@ public class UnitTest
 
     [Fact]
     public void TestRestrictedCreationOfCheepAt160Char(){
-        //Arrange
-        //Runs in memory
-        using var connection = new SqliteConnection("Filename=:memory:");
-        var builder = new DbContextOptionsBuilder<ChirpDBContext>();
-        builder.UseSqlite(connection);
-        var context = new ChirpDBContext(builder.Options);
-        context.Database.EnsureCreated();
-        var repository = new CheepRepository(context);
-        connection.Open();
-
         //Act
         CheepDTO cheep = new()
```

Comment:
Where is the assert

Reply:
The assert is on line 58, and is the act.Should()... method call.
See documentation here: https://fluentassertions.com/introduction